general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -715,3 +715,44 @@ def find_api_key() -> str:
         "Try passing an 'api_key' variable, or setting your API key in the command line "
         "with SUPERSTAQ_API_KEY=..."
     )
+
+
+def get_counts_on_qubits(
```

Comment:
At a high level, my biggest thoughts here are that this method should either
1. only return the counts on the requested qubits, or
2. be called something like `split_counts`.

I'm slightly leaning in favor of (1), because imo there isn't much gained from getting both sets of counts at the same time.

Reply:
So should I change it such that it fits the definition set by 1?